### PR TITLE
Rename donor list route to hall-of-fame

### PR DESCRIPTION
**Pull request by Abban**
https://phabricator.wikimedia.org/T274763

### CODE REVIEW

File: config/routes.yaml [hunk 1]
```diff
@@ -33,6 +33,14 @@ list-comments.html:
...
     controller: 'WMDE\Fundraising\Frontend\App\Controllers\Donation\ListCommentsController::handleHtml'
 
+Unterstützerliste-redirect:
+    path: '/page/Unterstützerliste'
+    methods: [GET]
+    controller: 'Symfony\Bundle\FrameworkBundle\Controller\RedirectController'
+    defaults:
+        path: '/page/hall-of-fame'
+        permanent: true
+
```
Comment (gbirke) [inline config/routes.yaml:43]: Please add a comment that this is temporary and needs to be removed when manitu has adapted the web sever config to do the redirect

We don't want the framework to do stuff a web server/load balancer can do more efficiently.

**Review by gbirke — APPROVED** [review]

**Review by gbirke — CHANGES_REQUESTED** [review]
For some reason, the composer update did not yet pick up the latest change to fundraising-frontend-content, leading to a 404 error when I tried it out. Updating the dependencies again made it work.

**Comment by moiikana** [comment]
when I just tested locally I also needed to run `make update-php`, else it leads to 404 and is still at "Unterstützerliste".
current state also deployed to here: testing02.wikimedia.customers.manitu.net

**Comment by gbirke** [comment]
I have moved the JS-related changes to https://gitlab.com/fun-tech/fundraising-app-frontend/-/merge_requests/16 

Please remove them from this PR

**Review by gbirke — APPROVED** [review]

**Pull request merged**